.github/workflows/AndroidCIWithGmd.yaml

```diff
@@ -0,0 +1,34 @@
+name: Android CI with GMD
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+
+  android-ci:
+    runs-on: macos-latest
+
+    steps:
+    - uses: actions/setup-java@v3
+      with:
+        distribution: 'temurin'
+        java-version: '11'
+    - uses: actions/checkout@v2
+
+    - name: Setup Android SDK
+      uses: android-actions/setup-android@v2
```

Comment:
Is this necessary? I don't see it in the other build scripts we have.

Reply:
Thanks. It wasn't necessary. The last attempt succeeded without this plugin.